README.md

```diff
@@ -351,6 +353,9 @@ player.eme.initializeMediaKeys(emeOptions, function(error) {
 });
 ```
 
+Note that if the callback returns anything other than `false`, an error will be fired on
```

Comment:
This behavior feels a bit funny to me. Would we be better off with an option passed to the function that specifies whether or not to fire an error on the player?

Reply:
I opted for this because it meant not having to change VHS: 
https://github.com/videojs/http-streaming/blob/master/src/videojs-http-streaming.js#L192

If you'd prefer a change to the function's API, and a PR to VHS, that's ok with me.

Reply:
I opted for this because it meant not having to change VHS. If you'd prefer a change to the function's API, and a PR to VHS, that's ok with me.

https://github.com/videojs/http-streaming/blob/846f5454c497fe61aa3f277ee8279d5ae2fe7252/src/videojs-http-streaming.js#L192

Reply:
We could just have a default value for the option, right? Or it gets tricky because of trying to properly position the callback within the params when adding a default?

Reply:
Yeah, I guess you're right that a default would be fine here.